Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -27,6 +27,26 @@ Before the testing library can be used, it must be added as a dependency of your
 Swift package or Xcode project. For more information on how to add it, see the
 [Getting Started](doc:TemporaryGettingStarted) guide.
 
+### Importing the testing library
+
+To import the testing library, add the following to the Swift source file that
+will contain the test:
+
+@Row {
+  @Column {
+    ```swift
+    // Before
+    import XCTest
+    ```
+  }
+  @Column {
+    ```swift
+    // After
+    import Testing
+    ```
+  }
+}
+
```

Comment:
For the purposes of documentation, it would be helpful if you include a brief paragraph here that explains that a test file can contain some tests written with XCTest and some tests written with swift-testing (which we refer to as "the testing library" in prose documentation.)

Reply:
I wrote it as a `- Note:` (similar to note in `DefiningTests.md`....).